pkgs/applications/terminal-emulators/ctx/default.nix

```diff
@@ -0,0 +1,58 @@
+{ lib
+, stdenv
+, fetchgit
+, SDL2
+, alsa-lib
+, babl
+, curl
+, libdrm # Not documented
```

Comment:
What do you mean with this comment?

Reply:
This dependency is required, but the documentation does not cite it.